### DISCUSSION TITLE
Two q-input components verify the association

### DISCUSSION THREAD

**Discussion by Solon-stray** [quasarframework/quasar#11791] [General - Components / Directives / etc] [answered]
For the correlation verification of two q-input components, but the first q-input requires two verification rules, and there is no way to perform responsive verificatio
[Uploading q-input.md…]()
So what should I do

**Comment by Solon-stray** [comment]
[q-input.md](https://github.com/quasarframework/quasar/files/7778201/q-input.md)

**Comment by MilosPaunovic** [comment]
Please create a Codepen reproduction of your issue so it would be easier for someone to play with it and provide you with better support.

**Comment by metalsadman** [comment, marked as answer]
`rules`only trigger when the input's model changes, what you could do is use a template ref for the first input then call `validate` manually in the related (second) input's events (update:model-value/blur etc..).

take heed of Milos's advice above next time.
https://codepen.io/metalsadman/pen/QWqaQPe?editors=1010
